python/ecl/__init__.py

```diff
@@ -111,12 +111,17 @@
     if not os.path.isdir( ecl_lib_path ):
         ecl_lib_path = None
 
-
 if sys.hexversion < required_version_hex:
     raise Exception("ERT Python requires Python 2.7.")
 
 def load(name):
-    return cwrapload(name, path=ecl_lib_path, so_version=ert_so_version)
+    try:
+        return cwrapload(name, path=ecl_lib_path, so_version=ert_so_version)
+    except ImportError:
+        # For pip installs, setup.py puts the shared lib in this directory
+        own_dir=os.path.dirname(os.path.abspath(__file__))
+        sys.stderr.write("Warning: Loading %s from path=%s failed. Trying to load from path=%s...\n" % (name, ecl_lib_path, own_dir))
```

Comment:
Remove the `stderr`message - this should be perfectly legitimate - no need to warn about this.